contrib/gardener/scripts/deploy-karydia-cp

```diff
@@ -222,7 +222,7 @@ spec:
       automountServiceAccountToken: false
       containers:
       - name: karydia
-        image: karydia/karydia:0.1.0
+        image: eu.gcr.io/gardener-project/karydia/karydia #TODO: add version number
```

Comment:
Does a comment work within the value part of a yaml list?

Reply:
In my understanding, this should not be a problem:
- Istio does it as well (https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/templates/endpoints.yaml#L12).
- The (documentation)[https://yaml.org/spec/1.2/spec.html#id2767100] does allow it, as long it is not in a scalar.
- The comments worked fine for me and did not throw any errors.

However, we can also change it to single line commands.